tests/flow/test_procedures.py

```
@@ -371,6 +371,24 @@ def test12_procedure_reordered_yields(self):
+    def test13_functions(self):
+        # Yield results of procedure in a non-default sequence
+        actual_resultset = self.graph.query("CALL dbms.functions() YIELD name RETURN name ORDER BY name").result_set
+        actual_len = len(actual_resultset)
+        self.env.assertEquals(actual_len, 95)
+
+    def test14_procedure_get_all_functions(self):
+        actual_resultset = self.graph.query("CALL dbms.functions() YIELD name, signature, description, return_description, argument_description RETURN name, signature, description, return_description, argument_description ORDER BY name").result_set
+        actual_len = len(actual_resultset)
+        self.env.assertEquals(actual_len, 95)
+    
+        expected_first_result = ["abs", 
```

Ruff

Local variable `expected_first_result` is assigned to but never used

Remove assignment to unused variable `expected_first_result`

(F841)